### PR TITLE
MHV-46798

### PR DESCRIPTION
**Pull request by gerard-sullivan**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary

MHV-46798

medications landing page content update based on the UCD

## Testing done

- _Describe what the old behavior was prior to the change_
- _Describe the steps required to verify your changes are working as expected_
- _Describe the tests completed and the results_
- _Exclusively stating 'Specs and automated tests passing' is NOT acceptable as appropriate testing
- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |   Screen Shot 2023-08-09 at 8 41 51 AM    |      Screen Shot 2023-08-09 at 9 19 46 AM
|
| Desktop |Screen Shot 2023-08-09 at 8 41 51 AM  | Screen Shot 2023-08-09 at 9 19 46 AM
 |
Screen Shot 2023-08-09 at 9 19 46 AM

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [X] Browser console contains no warnings or errors.
- [X] Events are being sent to the appropriate logging solution
- [X] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by acrollet — CHANGES_REQUESTED** [review]
Since this makes visual changes I'd love to see before & after screenshots per the PR template. Also, should any end to end tests be changed or added as part of this? Finally, it would be good to clean up the PR body to remove the boilerplate and make more clear what's changed and how a reviewer can test it. Thanks! 🙂

File: src/applications/mhv/medications/containers/LandingPage.jsx [hunk 1]
```diff
@@ -12,10 +12,7 @@ const LandingPage = () => {
         <div className="main-content">
           <section>
             <h1>About Medications</h1>
-            <p className="vads-u-font-size--h3">
-              Learn how to manage your VA prescriptions and review all
-              medications in your VA medical records.
-            </p>
+            <p className="vads-u-font-size--h3">Intro/lead text</p>
```
[the remark this answers is not in the document]
Comment (gerard-sullivan) [reply src/applications/mhv/medications/containers/LandingPage.jsx:15]: yes, right

File: src/applications/mhv/medications/containers/LandingPage.jsx [hunk 2]
```diff
@@ -37,7 +34,7 @@ const LandingPage = () => {
               Email your feedback and questions to us at <FeedbackEmail />.
             </p>
             <p>
-              Note: You still have access to the pharmacy tool on the My
+              <b>Note:</b> You still have access to the pharmacy tool on the My
```
Comment (acrollet) [inline src/applications/mhv/medications/containers/LandingPage.jsx:37]: Not blocking this PR but curious if this should be a strong tag? (Is it semantically important for screen reader users?)

**Review by acrollet — APPROVED** [review]

**Review by acrollet — APPROVED** [review]
@gerard-sullivan leaving an approval in case you need it to merge, there are a couple of linting errors and a failing test to fix.

**Pull request closed without merging**